packages/server/postgres/queries/getTeamsByOrgId.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  getTeamsByOrgIdQuery,
+  IGetTeamsByOrgIdQueryParams,
+  IGetTeamsByOrgIdQueryResult
+} from './generated/getTeamsByOrgIdQuery'
+import getPg from '../getPg'
+import catchAndLog from '../utils/catchAndLog'
+
+const getTeamsByOrgId = async (
+  orgIds: string | string[],
+  options: Partial<Omit<IGetTeamsByOrgIdQueryParams, 'orgId'>> = {}
+): Promise<IGetTeamsByOrgIdQueryResult[]> => {
+  orgIds = typeof orgIds === 'string' ? [orgIds] : orgIds
+  const orgs = await catchAndLog(() =>
+    getTeamsByOrgIdQuery.run(
+      {
+        orgIds,
+        ...options
```

Comment:
let's coerce `isArchived` to a boolean here. then we can get rid of the CAST + condition is the SQL.